### PR TITLE
Add Debian 10.9

### PR DESCRIPTION
**Pull request by mbautin**

### CODE REVIEW

File: docker_setup_scripts/docker_setup_scripts_common.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+yb_start_group() {
+  echo "echo ::group::$1"
+}
+
+yb_end_group() {
+  echo "::endgroup::"
+}
+
+yb_apt_get_dist_upgrade() {
```
Comment (jharveysmith) [inline docker_setup_scripts/docker_setup_scripts_common.sh:46]: This function seems unused

File: docker_setup_scripts/ubuntu.sh [hunk 2]
```diff
@@ -2,49 +2,48 @@
...
+  sudo
+  tzdata
+  unzip
+  uuid-dev
+  vim
+  wget
+  xz-utils
 )
 
 if [[ $ubuntu_major_version -le 18 ]]; then
```
Comment (jharveysmith) [inline docker_setup_scripts/ubuntu.sh:49]: This block could probably be removed too, our ubuntu docker base is 18.04

Comment (jharveysmith) [reply docker_setup_scripts/ubuntu.sh:49]: If we remove this, we could also remove line 9 and 10 above.

Comment (mbautin) [reply docker_setup_scripts/ubuntu.sh:49]: `-le` means less than or equal to. This is needed for Ubuntu 18.04.

**Review by jharveysmith — APPROVED** [review]

**Pull request merged**